.github/workflows/integration.yml

```diff
@@ -0,0 +1,81 @@
+name: Integration test with agoric-sdk
+
+on:
+  # Use the following to explicitly start thit workflow.
+  # scripts/start-integration-test.sh <BRANCH-OR-TAG>
+  workflow_dispatch:
+  # Otherwise, run on default branch.
+  push:
+    branches: [main] # $default-branch
+  pull_request: # Remove me before merging
```

Comment:
Just a reminder to remove this.

Reply:
Thanks for the reminder! Actually I think I want to run this as part of PRs, prevent merging.